src/main/kotlin/no/nav/tilleggsstonader/sak/vilkår/vilkårperiode/VilkårperiodeController.kt

```diff
@@ -35,21 +34,20 @@ class VilkårperiodeController(
         return vilkårperiodeService.hentVilkårperioder(behandlingId)
     }
 
-    @PostMapping("behandling/{behandlingId}")
+    @PostMapping
     fun opprettVilkårMedPeriode(
-        @PathVariable behandlingId: UUID,
-        @RequestBody opprettVilkårperiode: OpprettVilkårperiode,
+        @RequestBody lagreVilkårperiode: LagreVilkårperiode,
```

Comment:
kunne denne bare hett Vilkårperiode? (Selve variabelen) Evt. `nyVilkårperiode` her og `endretVilkårperiode` for oppdatering? Ser ut som et funksjonsnavn